components/up.libsonnet

```diff
@@ -60,13 +60,20 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
     local c =
       container.new('observatorium-up', up.config.image) +
       container.withArgs([
-        '--endpoint-read=' + up.config.readEndpoint,
-        '--duration=0',
-        '--queries-file=/etc/up/queries.yaml',
-        '--log.level=debug',
-      ]) + container.withPorts([
+                           '--duration=0',
```

Comment:
nit: I don't think this indentation changes necessary. Better to organize the code so that formatted wouldn't change it in such a way.

Reply:
good catch, thanks, I didn't even realize it looks to messed up.


I think it is much better now